libs/git_wrapper/Git_wrapper.mli

```diff
@@ -16,6 +16,24 @@ type status = {
 *)
 val files_from_git_ls : cwd:Fpath.t -> Fpath.t list
 
+(* Executing a function inside a directory created from git-worktree.
+
+   `git worktree` is doing 90% of the heavy lifting here. Docs:
```

Comment:
maybe this comment needs to be updated. 90% of which lifting?
I think this comment should be put next to the code that use git-worktree for the baseline problem.